config/core.js

```diff
@@ -122,6 +114,25 @@ config.SESSION_OPTIONS = {
 for (var attr in process.env) {
 	if (attr && attr.startsWith('NPOMF_')) {
 		eattr = attr.replace('NPOMF_', '');
-		config[eattr] = process.env[attr];
+    
+    var data = process.env[attr];
+    // lets make it possible to use stuff like CONTACTS in docker environments.
+    try {
+      config[eattr] = JSON.parse(data);
+    } catch (exception) {
+      config[eattr] = data;
+    }
 	}
 }
+
+// lets move it here so changes made to MAX_UPLOAD_SIZE will be displayed correctly.
+if (typeof config.DESCRIPTION == 'undefined') {
```

Comment:
I prefer `=== undefined` rather than `typeof == 'undefined'`, probably best to check null as well.